src/examples/src/widgets/constrained-input/Username.tsx

```diff
@@ -0,0 +1,26 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import ConstrainedInput from '@dojo/widgets/constrained-input';
+import icache from '@dojo/framework/core/middleware/icache';
+
+const factory = create({ icache });
+
+export default factory(function Basic({ middleware: { icache } }) {
+	const value = icache.getOrSet('value', '');
+
+	return (
+		<ConstrainedInput
+			rules={{
+				length: {
+					min: 4,
```

Comment:
i think this should still include special chars / uppercase etc to show the usage

Reply:
Maybe i'll rename it to something that isn't username related then, just something real generic? I don't think it makes sense for a username example to have to include special characters.

Reply:
Ended up just adding those rules back 👍